app/src/main/java/org/openhdp/hdt/ui/tracking/TrackingItem.kt

```diff
@@ -4,27 +4,35 @@ import androidx.annotation.ColorInt
 
 
 data class TrackingItem(
-    val id: String,
+    val stopWatchId: String,
```

Comment:
I think it should be stopwatchId.

Reply:
it's UI model, I promise to change it in upcoming PR 🤞